ci-operator/config/codeready-toolchain/registration-service/codeready-toolchain-registration-service-master.yaml

```diff
@@ -0,0 +1,54 @@
+
+base_images:
+  os:
+    name: centos
+    namespace: openshift
+    tag: '7'
+
+resources:
+  '*':
+    limits:
+      memory: 3Gi
+    requests:
+      memory: 250Mi
+      cpu: 100m
+promotion:
+  namespace: codeready-toolchain
+  name: registration-service-v0.1
+images:
+- from: os
+  inputs:
+    bin:
+      paths:
+      - destination_dir: .
+        source_path: /go/src/github.com/codeready-toolchain/registration-service/build/_output/bin/registration-service
+  dockerfile_path: openshift-ci/Dockerfile.deploy
+  to: registration-service
+    
+build_root:
+  project_image:
+    dockerfile_path: openshift-ci/Dockerfile.tools
+
+binary_build_commands: make build
+
+tag_specification:
+  name: "4.2"
+  namespace: ocp
+
+tests:
+- as: build
+  commands: make build
+  container:
+    from: src
+- as: lint
+  commands: make lint-yaml
+  container:
+    from: src
+- as: test
+  commands: make test-with-coverage upload-codecov-report
```

Comment:
Make sure to add `test-with-coverage` and `upload-codecov-report` make targets in registration-service repository

Reply:
Adding with PR https://github.com/codeready-toolchain/registration-service/pull/51